tests/test_stamp.py

```diff
@@ -40,7 +40,7 @@ def test_stamp_self_join(T_A, T_B):
     zone = int(np.ceil(m / 2))
     left = np.array(
         [
-            naive_mass(Q, T_B, m, i, zone)
+            utils.naive_mass(Q, T_B, m, i, zone, ignore_trivial=True)[0:2]
```

Comment:
I don't like the indexing `[0:2]` here. `left` should be raw and untouched. Move the indexing to the assertion so that it is more explicit/obvious as to what we are comparing. Since most other places are comparing all of matrix profille and indices, it would be more consistent to have it in the assertion

Reply:
I thought that by having [0:2] there we would have more readability. naive mass also returns the left and right indices, something that STAMP does not. Before there was a dedicated naive_mass function in test_stamp, but I don't like having the same function in two different places